### ISSUE TITLE
Valetudo is not running

### ISSUE THREAD

**Issue by lchimaru** [rumpeltux/viomi-rooting#8]
I have rooted my vacuum, but Valetudo seems to not starting on boot. I have checked files created during installation process, and all of them are present and looks good. But Valetudo is not running on startup, web interface is not reachable and I'm able to see vacuum in my Xiaomi Home app still. Any ideas what could went wrong?

**Comment by lchimaru** [comment]
For anyone facing same issue - manual Valetudo installation helped.

**Comment by rumpeltux** [comment]
@lchimaru do you have any insight what was missing / different so that we can fix the installation script? Thank you!

**Comment by CoryManson** [comment]
Hi @rumpeltux, I've just experienced the same issue. I ended up doing a manual isntallation but did check over your script and it had done everything correctly.

It appears the execute flag was missing on the binary so the service couldn't be started.

![image](https://user-images.githubusercontent.com/5052021/93840818-1217ba80-fcd5-11ea-96c1-9e5c1a4b6505.png)

![image](https://user-images.githubusercontent.com/5052021/93840812-075d2580-fcd5-11ea-8795-9cc04a5a988c.png)
